bindata/network/openshift-sdn/000-ns.yaml

```diff
@@ -9,3 +9,16 @@ metadata:
   annotations:
     openshift.io/node-selector: "" #override default node selector
     openshift.io/description: "OpenShift SDN components"
+---
```

Comment:
I feel like this should probably be a separate YAML... the main `openshift-sdn` namespace is `000-ns.yaml` because it has to be created first, before anything else in that namespace. But the host network namespace is a totally separate thing. It can be in its own file, and doesn't need to be created in any particular order. (So just, like, `host-network-ns.yaml`)